connect/cli/plugins/project/report/templates/bootstrap/project_dir/HOWTO.md.j2

```diff
@@ -0,0 +1,16 @@
+# Welcome to Connect Extension project {{ project_name }}
```

Comment:
AFAIK this file is new. If you want to include it please fix the content.
It's a report project not `Extension`.
We do nout use docker, so please fix the next steps. You can explain how the generate function of the entrypoint must be written, how to validate, how to run pytest and how to execute the report (locally).